openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -68,6 +68,8 @@ class AccountView(APIView):
 
                 * goals: null or textual representation of goals
 
+                * bio: null or textural representation of user biographical information ("about me")
```

Comment:
Should there be any restrictions on what can go in the bio? I assume that any Unicode character is valid. Should we say that HTML is not supported (as in it will be escaped into the bio to avoid script injection). Did we decide that there is essentially no size limit?